source/style-guide/style/code-examples.txt

```diff
@@ -29,21 +30,20 @@ or output examples:
 -  When the command is simple, and there's nothing specific to say
    about the output, you can show the input and output in the same code
    block, as users would actually see the code in their own terminal.
-   The inclusion of the command prompt differentiates the input from
-   the output.
+   In this case, you can include the command prompt to differentiate the
+   input from the output. Ensure that this code block is set to 
+   ``:copyable: false``.
```

Comment:
Would it be useful to also include to mention that a command should be run as root or elevated user prior to the command? It might not be clear to all audiences that "#" signifies this.